kickoff/views/releases.py

```diff
@@ -96,6 +96,70 @@ def post(self, releaseName):
         db.session.commit()
         return Response(status=200)
 
+class ReleasesListAPI(MethodView):
+    def getTotal(self, complete = None, ready = None):
+        filter = {}
+
+        if complete is not None:
+            filter["complete"] = complete
+
+        if ready is not None:
+            filter["ready"] = ready
+
+        return ProductReleasesView.query.filter_by(**filter).count()
+
+    def getOrderByDict(self):
+        #Warning: This is a server-side function that is highly dependent of Jquery Datatables
```

Comment:
Could you add a check on the version of jquery ? we could show a warning in the console when it differs.
Otherwise, I am sure we will spend hours to debug this!